.github/workflows/actions.yml

```diff
@@ -0,0 +1,25 @@
+name: Linter Actions Workflow
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
```suggestion
      - main
```

Reply:
I was just typing to ask why the action isn't running...